src/sls.ts

```diff
@@ -0,0 +1,5 @@
+const completionSpec: Fig.Spec = {
+  name: "sls",
+  loadSpec: "serverless",
+};
```

Comment:
This is more performant:

```suggestion
import serverless from "./serverless";
const completionSpec: Fig.Spec = {
  ...serverless,
  name: "sls",
};
```